opentreemap/importer/views.py

```diff
@@ -110,6 +113,18 @@ def get_import_table(request, instance, table_name):
 _EVENT_TABLE_PAGE_SIZE = 5
 
 
+def _cleanup_tables(instance):
+    ievents = (list(TreeImportEvent.objects.filter(instance=instance)) +
+               list(SpeciesImportEvent.objects.filter(instance=instance)))
+    for ie in ievents:
+        if ie.task_id != '':
+            result = AsyncResult(ie.task_id)
+            if result.failed():
```

Comment:
Do we also want to mark the import as failed if we could not retrieve the `AsyncResult`?  I'm not sure how long Celery/Redis holds onto these, and I doubt they'd still be there after switching Redis backends, like we've recently done.